source/extensions/filters/http/cache/simple_http_cache/simple_http_cache.cc

```diff
@@ -30,9 +32,9 @@ class SimpleLookupContext : public LookupContext {
     cb(std::make_unique<Buffer::OwnedImpl>(&body_[range.begin()], range.length()));
   }
 
-  void getTrailers(LookupTrailersCallback&&) override {
-    // TODO(toddmgreer): Support trailers.
-    NOT_IMPLEMENTED_GCOVR_EXCL_LINE;
+  void getTrailers(LookupTrailersCallback&& cb) override {
+    cb(trailers_ ? Http::createHeaderMap<Http::ResponseTrailerMapImpl>(*trailers_)
```

Comment:
There's no need to create a new ResponseTrailerMap. `cb(std::move(trailers_))` is simpler and more efficient.

Reply:
Thanks for your suggestion, Todd!